ext/data/schemas/recommended-dictionaries-schema.json

```diff
@@ -0,0 +1,100 @@
+{
+    "$id": "recommendedDictionaries",
```

Comment:
Is the schema just for documentation, i.e. it will not be used for validation anywhere?

Reply:
It is validated in the json tests